.github/dependabot.yml

```diff
@@ -25,3 +25,10 @@ updates:
     # Check for updates once a week
     schedule:
       interval: "daily"
+
+  # Enable version updates for pip
+  - package-ecosystem: "pip"
+    directory: "/"

```

Comment:
We have two python modules in this repo:
- one under `client/python`
- one under `third_party/airflow`

IMBW, but I believe this will have to be pointed at the module dirs directly (where the pyproject.toml is) in order for this to be effective.

Reply:
I made the required changes. Unfortunately, it's still impossible to add multiple dirs into the same package ecosystem.
Here's what I found on this: 
https://github.com/dependabot/dependabot-core/issues/2824
https://github.com/iZettle/example-integration/pull/7

However, I think that my version was correct - it was should have been checking all python packages in the whole root dir. At least accrording to what I've read so far. IMBW, too, of course. It's something for me to learn more about, for sure!

---
I was looking for some open source projects I could contribute to for the first time and to practice using GitHub, and this way I found Armada.
I'm actually trying to learn more about Armada itself rn. : )

Reply:
I think the revision is better; I prefer being explicit about where to check. 

Thanks for coming by! If you have any questions, feel free to pop them into github discussions. Otherwise, if you continue to be interested in contributing, please just pick up an untouched issue and comment on it. Those of us working full time on Armada work right out of github issues, so that should be up to date.